Integration/AbstractEnhancerIntegration.php

```diff
@@ -70,14 +65,10 @@ public function buildEnhancerFields()
                 }
 
                 $this->fieldModel->saveEntity($new_field);
-                if (!in_array($alias, $created)) {
-                    $created[] = $alias;
-                }
+                $last_field = $new_field;
             }
-
-            $feature_settings['installed'] = $created;
-            $integration->setFeatureSettings($feature_settings);
         }
+        $this->fieldModel->reorderFieldsByEntity($last_field);
```

Comment:
`$last_field` may not always be set, as is not necessary since it will always be either `undefined` or the same as `$new_field`